libs/candles-chart/src/lib/candles-chart.tsx

```diff
@@ -168,6 +167,11 @@ export const CandlesChartContainer = ({
             chartType: chartType,
             overlays: overlays,
             studies: studies,
+            notEnoughDataText: (
```

Comment:
Could you please update the account history chart as well? That uses the PriceChart

Reply:
Unfortunately this type of charts has been omitted, and we don't have possibility now to set this param there as reactNode :-). I don't want to hold longer this solution, so I created a new issue for fixing it: https://github.com/vegaprotocol/frontend-monorepo/issues/2836